src/main/scala/GuessWho/GuessWho.scala

```diff
@@ -15,29 +17,60 @@ object GuessWho extends App{
 
   private val characterList:Seq[Character] = Seq(jon, tony, bill, paul, gary, steve, jess, vicky, sarah, beff)
 
+
   private val game:GameBoard = new GameBoard(characters = characterList, defaultChosenCharacter = None)
+  private val interface: Interface = new Interface()
+
 
   private var _playersRemaining:Seq[Character] = game.getRemainingCharacters
+  private var _newTurn:Boolean = true
```

Comment:
Having underscore infront of values isn't a Scala thing, I think this is Python style?